tck/api/src/main/java/org/eclipse/microprofile/metrics/tck/TagsTest.java

```diff
@@ -22,53 +22,168 @@
+        Tag tagRed = new Tag("colour", "red");
+        Tag tagBlue = new Tag("colour", "blue");
+        
+        String counterName = "org.eclipse.microprofile.metrics.tck.TagTest.counterColour";
+        
+        Counter counterColour = registry.counter(counterName);
+        Counter counterRed = registry.counter(counterName,tagEarth,tagRed);
+        Counter counterBlue = registry.counter(counterName,tagEarth,tagBlue);
+        
+        MetricID counterColourMID = new MetricID(counterName);
+        MetricID counterRedMID = new MetricID(counterName, tagEarth,tagRed);
+        MetricID counterBlueMID = new MetricID(counterName, tagEarth,tagRed);
+        
+        //check multi-dimensional metrics are registered
+        assertThat("Counter is not registered correctly", registry.getCounters(), hasKey(counterColourMID));
+        assertThat("Counter is not registered correctly", registry.getCounters(), hasKey(counterRedMID));
+        assertThat("Counter is not registered correctly", registry.getCounters(), hasKey(counterBlueMID));
+    }
+    
+    @Test
+    @InSequence(4)
+    public void meterTagsTest() {
+        
+        Tag tagEarth = new Tag("planet", "earth");
+        Tag tagRed = new Tag("colour", "red");
+        Tag tagBlue = new Tag("colour", "blue");
+        
+        String meterName = "org.eclipse.microprofile.metrics.tck.TagTest.meterColour";
+        
+        Meter meterColour = registry.meter(meterName);
+        Meter meterRed = registry.meter(meterName,tagEarth,tagRed);
+        Meter meterBlue = registry.meter(meterName,tagEarth,tagBlue);
+        
+        MetricID meterColourMID = new MetricID(meterName);
+        MetricID meterRedMID = new MetricID(meterName, tagEarth,tagRed);
+        MetricID meterBlueMID = new MetricID(meterName, tagEarth,tagRed);
+        
+        //check multi-dimensional metrics are registered
+        assertThat("Meter is not registered correctly", registry.getMeters(), hasKey(meterColourMID));
+        assertThat("Meter is not registered correctly", registry.getMeters(), hasKey(meterRedMID));
+        assertThat("Meter is not registered correctly", registry.getMeters(), hasKey(meterBlueMID));
+    }
+    
+    @Test
+    @InSequence(5)
+    public void timerTagsTest() {
+        
+        Tag tagEarth = new Tag("planet", "earth");
+        Tag tagRed = new Tag("colour", "red");
+        Tag tagBlue = new Tag("colour", "blue");
+        
+        String timerName = "org.eclipse.microprofile.metrics.tck.TagTest.timerColour";
+        
+        Timer timerColour = registry.timer(timerName);
+        Timer timerRed = registry.timer(timerName,tagEarth,tagRed);
+        Timer timerBlue = registry.timer(timerName,tagEarth,tagBlue);
+        
+        MetricID timerColourMID = new MetricID(timerName);
+        MetricID timerRedMID = new MetricID(timerName, tagEarth,tagRed);
+        MetricID timerBlueMID = new MetricID(timerName, tagEarth,tagRed);
```

Comment:
think this last one was supposed to say...
MetricID counterBlueMID = new MetricID(counterName, tagEarth,tagBlue);